src/eventSchemas/commands/downloadSchemaItemCode.ts

```diff
@@ -313,27 +322,53 @@ export class CodeExtractor {
         return tempFolder
     }
 
-    // Ensure that the downloaded code hierarchy has no collisions with the destination directory
-    public validateNoFileCollisions(codeZipFile: string, destinationDirectory: string): void {
+    // Check if downloaded code hierarchy has collisions with the destination directory and display them in output channel
+    public checkFileCollisions(
+        codeZipFile: string,
+        destinationDirectory: string,
+        outputChannel: vscode.OutputChannel
+    ): boolean {
         const zip = new admZip(codeZipFile)
         const zipEntries = zip.getEntries()
+        let collisionExist = false
 
         zipEntries.forEach(function(zipEntry) {
             if (zipEntry.isDirectory) {
                 // Ignore directories because those can/will merged
             } else {
                 const intendedDestinationPath = path.join(destinationDirectory, '/', zipEntry.entryName)
                 if (fs.existsSync(intendedDestinationPath)) {
-                    throw new UserNotifiedError(
-                        localize(
-                            'AWS.message.error.schemas.downloadCodeBindings.timeout',
-                            'Unable to place schema code in workspace because there is already a file {0} in the folder hierarchy',
-                            zipEntry.name
+                    if (!collisionExist) {
+                        collisionExist = true
+                        outputChannel.show(true)
+                        outputChannel.appendLine(
+                            localize(
+                                'AWS.message.info.schemas.downloadCodeBindings.colliding_files',
+                                'Following files already exist in the folder hierarchy :'
+                            )
                         )
-                    )
+                    }
+                    outputChannel.appendLine(intendedDestinationPath)
                 }
             }
         })
+
+        return collisionExist
+    }
+
+    public async overrideOnCollisionUserResponse(): Promise<boolean> {
```

Comment:
function name is confusing. Consider something like:
* promptToOverwriteCodeBindings
* confirmOverwriteCollisions